barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -446,15 +446,25 @@ void gateCount(const std::string& bytecodePath, bool honk_recursion)
     size_t i = 0;
     for (auto constraint_system : constraint_systems) {
         acir_proofs::AcirComposer acir_composer(0, verbose_logging);
-        acir_composer.create_circuit(constraint_system);
+        acir_composer.create_circuit(constraint_system, {}, true);
```

Comment:
A question also for other reviewers: in general I recommend adding `/*param=*/{}` for any literal value or when the param name is not clear. This does wonders for readability (e.g. here, reviewing). However, VS already fills in the name for you.

What's the recommendation? (from my side, I think I still prefer adding the comment)

Reply:
I typically don't do it for languages that have language servers that provide this info, but as you guys prefer